src/CruxComponent.tsx

```diff
@@ -75,7 +76,7 @@ export class CruxComponentCreator {
 
```

Comment:
wont pagination break foreign models ?

Reply:
Not it will not break, for Foreign Model, it will be same as earlier.
We have checked it